kivy/uix/scatter.py

```diff
@@ -244,8 +246,10 @@ def _get_bbox(self):
             if y > ymax:
                 ymax = y
         return (xmin, ymin), (xmax - xmin, ymax - ymin)
-    bbox = AliasProperty(_get_bbox, None, bind=(
-        'transform', 'width', 'height'))
+
+    bbox = AliasProperty(_get_bbox,
```

Comment:
It would work for `Scatter` instances only, but if `Scatter` is sub-classed and `to_parent` changed then user would have to re-cache the value. 
I'll remove it in next commit.